test/stdlib/LibcFeatureFlag.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %target-typecheck-verify-swift
+// REQUIRES: executable_test
```

Comment:
Since we're not actually executing the test, there's no need for this anymore…

Reply:
I can kill it but I was hoping to leave it document the inconsistency/differences with glibc coverage depending on the platform you run the test on (like this test will pass on FreeBSD and Android but not Linux) until we can figure out what feature flag we want to set. Once we add a feature flag for Linux this test should start passing and we can kill the XFAIL :smile:

Reply:
:-/ "REQUIRES: executable_test" doesn't distinguish between Linux and FreeBSD, though.

Reply:
Gotcha. I'll kill the test until we figure out how to enable this flag for Linux eventually.

Reply:
As I understand it you want something like `REQUIRES: executable_test` and `UNSUPPORTED: OS=linux-gnu`.

Reply:
(`XFAIL` always runs the test, but expects the test to fail on the specified configurations. `UNSUPPORTED` skips the test on the specified configurations.)